charts/ceph-csi-rbd/values.yaml

```diff
@@ -217,6 +217,107 @@ topology:
     - failure-domain/region
     - failure-domain/zone
 
+storageClass:
+  # Specifies whether the Storage class should be created
+  create: true
+  name: csi-rbd-sc
+  # String representing a Ceph cluster to provision storage from.
+  # Should be unique across all Ceph clusters in use for provisioning,
+  # cannot be greater than 36 bytes in length, and should remain immutable for
+  # the lifetime of the StorageClass in use.
+  clusterID: <cluster-ID>
+  # Ceph pool into which the RBD image shall be created
+  # eg: pool: replicapool
+  pool: <pool-name>
+  # RBD image features, CSI creates image with image-format 2
+  # CSI RBD currently supports only `layering` feature.
+  imageFeatures: layering
+
+  # (optional) uncomment the following to use rbd-nbd as mounter
+  # on supported nodes
+  # mounter: rbd-nbd
+  mounter: ""
+
+  # (optional) Prefix to use for naming RBD images.
+  # If omitted, defaults to "csi-vol-".
+  # volumeNamePrefix: "foo-bar-"
+  volumeNamePrefix: ""
```

Comment:
same here also comment out the  optional parameters

Reply:
The values file should contain the value and left empty if un-used (So that the template can read that the value is empty and not use the key pair while deploying). Since the `volumeNamePrefix` parameter is optional, it is kept empty; similar to the convention used throughout the file.
Let me know if any changes are required :)